Cargo.toml

```diff
@@ -311,9 +311,10 @@ flate2 = "1.0"
 serde = { version = "1", features = ["derive"] }
 bytemuck = "1.7"
 # Needed to poll Task examples
-futures-lite = "2.0.1"
```

Comment:
Did you purposefully remove `futures-lite`, or was this a mistake?

Reply:
I did it on purpose. Based on git history, I see that before #10675 there were multiple versions used at the same time, and then it was streamlined to use the same version everywhere. But I don't see reason to specify that here, since it is also a regular dependency. Am I wrong here?

Reply:
After a bit of thinking I decided to remove that change and make a separate PR with some deps cleanup